spring-cloud-gcp-pubsub/src/main/java/org/springframework/cloud/gcp/pubsub/support/DefaultSubscriberFactory.java

```diff
@@ -156,12 +159,22 @@ public void setApiClock(ApiClock apiClock) {
 	/**
 	 * Set the retry settings for the generated subscriber stubs.
 	 * @param subscriberStubRetrySettings parameters for retrying pull requests when they fail,
-	 * including jitter logic, timeout, and exponential backoff
+	 *                                    including jitter logic, timeout, and exponential backoff
```

Comment:
Although this makes a Javadoc more readable, it doesn't look like a necessary change.
Might be just an auto-refactoring artifact in the IDE...

Reply:
Fixed - IDE issue.